### DISCUSSION TITLE
Root Password for LBB user 'lbb-desktop'

### DISCUSSION THREAD

**Discussion by keeratsawhney** [outdoorbits/little-backup-box#340] [Q&A]
I am trying to Sudo cd, It is asking for Password for lbb-desktop. I am not sure it is creating two users One is User 'Pi' other is User 'lbb-desktop'. It has a password which is never set. Could you help with this. I did a fresh Installation just to see if the password vanishes but no success. It still has password.

![20241213_164735.jpg](https://github.com/user-attachments/assets/f931cada-c4f7-4ea1-b909-a612dc85fa1c)

**Comment by outdoorbits** [comment]
Please try to login as user pi using ssh.
